app/helpers/quadicon_helper.rb

```diff
@@ -214,15 +214,15 @@ def render_quadicon_label(item, row)
     opts    = quadicon_build_label_options(item, row)
 
     if quadicon_hide_links?
-      content_tag(:span, content, opts)
+      content_tag(:span, content, opts[:options])
     else
       url = quadicon_build_label_url(item, row)
       quadicon_link_to(url, **opts) { content }
     end
   end
 
   # FIXME: Even better would be to ask the object what name to use
-  def quadicon_label_content(item, row)
+  def quadicon_label_content(item, row, truncate: true)
```

Comment:
Oops, didn't pay attention and thought you referring to the calling code. (what Rubocop complained about) Here it has to stay this way because that's the syntax for a keyword-arg default value.